packages/hub/tsconfig.json

```diff
@@ -10,7 +10,7 @@
     "esModuleInterop": true,
     "strict": false
   },
-  "references": [{"path": "../devtools"}],
+  "references": [{"path": "../devtools"}, {"path": "../nitro-protocol"}],
```

Comment:
Do we need to enable `composite` setting in `nitro-protocol` and `devtools` for references to work properly? https://www.typescriptlang.org/docs/handbook/project-references.html#composite

Reply:
Yeah, and we do. It's in the root `tsconfig.json`.

Reply:
Ok yep, that makes sense.